### ISSUE TITLE
ggVennDiagram memory vs ggvenn

### ISSUE THREAD

**Issue by dy-lin** [gaospecial/ggVennDiagram#19]
Using `ggVennDiagram_0.5.0`:

My dataset:
```
Large list (3 elements, 23.5 MB)
List of 3
 $ Score : chr [1:129544] 
 $ Length: chr [1:5109]
 $ Charge: chr [1:88312]
```

I'm trying to use `ggVennDiagram` on the dataset listed above. However, it seems that it uses a lot of memory, and R crashes with 8GB of RAM. I was able to subsample and run with a smaller dataset (see below), so I know the program works.

```
Large list (3 elements, 3.5 MB)
List of 3
 $ Score : chr [1:19016] 
 $ Length: chr [1:793]
 $ Charge: chr [1:13012]
```

![Rplot](https://user-images.githubusercontent.com/44071582/110995541-5ce2c880-832f-11eb-9966-e4516dd8e164.png)

I am able to run the same dataset using `ggvenn` (no memory issues, see figure below) but would prefer to use `ggVennDiagram` due to the counts legend.

![Rplot01](https://user-images.githubusercontent.com/44071582/110995736-b1864380-832f-11eb-8588-ca589e32053d.png)

Is there a way to reduce the memory usage of the program, or any other work around?

**Comment by gaospecial** [comment]
Thanks for your feedback. I can reproduce this failure in my R session runing on a 16 GB memory PC. I'll let you know if find a resolution.

**Comment by gaospecial** [comment]
@dy-lin I have released the 1.0 version ggVennDiagram. My test shows this version is no problem with large list input. You may try it and give me any feedbacks. Thank you.
